### PR TITLE
adds pattern for minimizing deployed environments

### PR DESCRIPTION
**Pull request by camcash17**
A proposal for minimizing deployed environments as green patterns

Re-opened from #26

### CODE REVIEW

File: docs/catalog/minimizing-deployed-environments.md [hunk 1]
```diff
@@ -0,0 +1,46 @@
+---
+version: 1.0
+submitted_by: @camcash17
+published_date: TBD
+category: cloud
+tags: 
+ - cloud
+ - environments
```
Comment (dubrie) [inline docs/catalog/minimizing-deployed-environments.md:8]: We can only support tags from the [suggested tags](https://patterns.greensoftware.foundation/guide/suggested-tags) list. If there are tags you want to include that aren't listed there, you can open an issue to propose the new ones.

For this pattern, I think the following tags would make the most sense:
- cloud
- deployment

File: docs/catalog/minimizing-deployed-environments.md [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+ - development
+ - performance
+ - qa
+---
+
+# Minimising the total number of deployed environments
+
+## Description
+
+From an energy-efficiency perspective, it's better to minimise the amount of deployed environments for an application so that less energy is required because there is less resources provisioned.
```
Comment (dubrie) [inline docs/catalog/minimizing-deployed-environments.md:19]: This fits better in the solution section below. The description can focus more on the problem itself and providing context on why or how deployment environments can grow.

**Review by dubrie — CHANGES_REQUESTED** [review]
The pattern looks good, some small tag clean-up changes and then think about a description that provides more information and and context and doesn't focus on the solution.

**Comment by dubrie** [comment]
Thanks for the updates @camcash17! Just some small changes left and we can submit.

File: docs/catalog/minimizing-deployed-environments.md (hunk 2 above)
Comment (camcash17) [reply docs/catalog/minimizing-deployed-environments.md]: Updated this description. Please let me know if this works better!

**Comment by dubrie** [comment]
Love it, thanks @camcash17!

**Pull request merged**